corehq/apps/reports/analytics/sqlaccessors.py

```diff
@@ -0,0 +1,26 @@
+from django.db.models.aggregates import Count
+
+from corehq.apps.sofabed.models import FormData
+
+
+def get_form_counts_for_user_by_date(user_ids, startdate, enddate, date_field, timezone):
```

Comment:
also, both of these functions should definitely have domain-level filtering applied, no? is that a bug in the existing code?